src/components/FormSchedule.vue

```diff
@@ -78,31 +88,39 @@
 </template>
 
 <script>
+
 import DatePicker from "vue2-datepicker";
 import { ACCESS_TOKEN } from "../constants/index.js";
 import axios from "axios";
-import { API_BASE_URL } from "../constants";
+import { API_BASE_URL,USER_LANGUAGE,CREATESCHEDULE,SAVE,CANCEL,REPEAT,ACTIVE,MESSAGE,RUNAT,SOMECHANNELNAME,SOMEMESSAGETITLE,NEWMESSAGE,TYPEYOURMESSAGE } from "../constants";
 const headers = {
   "Content-Type": "application/json",
   Authorization: "Bearer " + localStorage.getItem(ACCESS_TOKEN)
 };
 
+
 export default {
   name: "formaS",
   components: { DatePicker },
   data() {
     return {
       messagesData: [],
       channelsData: [],
-      messageTitle: "Some message title",
-      channelName: "Some channel name",
+
+      messageTitle:  localStorage.getItem(
```

Comment:
This also might be nice way to go for translations: Set values of translations to component property, and bind it in template. (Refering to my comment up)